pattern-lab/install.yml

```diff
@@ -0,0 +1,11 @@
+##
+# Container for the initial install
+##
+version: '2.1'
+
+services:
```

Comment:
They really could be combined (in which case I would return to the build.yml style naming). I was originally looking to break out things which run to completion automatically from things which should run interactively. That distinction and the proliferation of yaml files seems to fall a little flat now that you've prompted me to consider it more. I'll update to reconsolidate into a build.yml file.